geode-core/src/integrationTest/java/org/apache/geode/internal/cache/TXDetectReadConflictJUnitTest.java

```diff
@@ -69,7 +81,7 @@ protected void createCachePR() {
     props.put(MCAST_PORT, "0");
```

Comment:
These uses of `Properties.put` are pre-existing but should be changed to `Properties.setProperty` if you're up for it.